quantum/qmk_settings.c

```diff
@@ -48,16 +48,16 @@ static const qmk_settings_proto_t protos[] PROGMEM = {
    DECLARE_STATIC_SETTING(6, osk_timeout),
    DECLARE_STATIC_SETTING(7, tapping_term),
    DECLARE_STATIC_SETTING(8, tapping),
-#ifdef MOUSEKEY_ENABLE
-   DECLARE_STATIC_SETTING_NOTIFY(9, mousekey_delay, mousekey_apply),
-   DECLARE_STATIC_SETTING_NOTIFY(10, mousekey_interval, mousekey_apply),
-   DECLARE_STATIC_SETTING_NOTIFY(11, mousekey_move_delta, mousekey_apply),
-   DECLARE_STATIC_SETTING_NOTIFY(12, mousekey_max_speed, mousekey_apply),
-   DECLARE_STATIC_SETTING_NOTIFY(13, mousekey_time_to_max, mousekey_apply),
-   DECLARE_STATIC_SETTING_NOTIFY(14, mousekey_wheel_delay, mousekey_apply),
-   DECLARE_STATIC_SETTING_NOTIFY(15, mousekey_wheel_interval, mousekey_apply),
-   DECLARE_STATIC_SETTING_NOTIFY(16, mousekey_wheel_max_speed, mousekey_apply),
-   DECLARE_STATIC_SETTING_NOTIFY(17, mousekey_wheel_time_to_max, mousekey_apply),
+#if defined(MOUSEKEY_ENABLE) && !defined(MK_3_SPEED)
+    DECLARE_STATIC_SETTING_NOTIFY(9, mousekey_delay, mousekey_apply),
+    DECLARE_STATIC_SETTING_NOTIFY(10, mousekey_interval, mousekey_apply),
+    DECLARE_STATIC_SETTING_NOTIFY(11, mousekey_move_delta, mousekey_apply),
+    DECLARE_STATIC_SETTING_NOTIFY(12, mousekey_max_speed, mousekey_apply),
+    DECLARE_STATIC_SETTING_NOTIFY(13, mousekey_time_to_max, mousekey_apply),
+    DECLARE_STATIC_SETTING_NOTIFY(14, mousekey_wheel_delay, mousekey_apply),
+    DECLARE_STATIC_SETTING_NOTIFY(15, mousekey_wheel_interval, mousekey_apply),
+    DECLARE_STATIC_SETTING_NOTIFY(16, mousekey_wheel_max_speed, mousekey_apply),
+    DECLARE_STATIC_SETTING_NOTIFY(17, mousekey_wheel_time_to_max, mousekey_apply),
 #endif
```

Comment:
There's still whitespace change here that needs to be removed

Reply:
Oh.. i will do it right away.